### PR TITLE
Remove clj-yaml as a runtime dependency.

### PR DESCRIPTION
**Pull request by vigneshsarma**
- clj-yaml depends on snakeyaml which has a vauln. clj-yaml has yet to merge the fix.
- Looking at the code there is not real reason to keep the clj-yaml if minor build update is added.
- Run `cat src/resources/submodules/regexes.yaml | jet -i yaml -o edn -k > resources/regexes.edn`
every time the submodule is updated.

### CODE REVIEW

File: resources/regexes.edn
Comment (vigneshsarma) [inline resources/regexes.edn:1]: Let me know if this should be just deleted. I could be an ignored file that is just available during build.

File: README.md [hunk 1]
```diff
@@ -18,6 +18,15 @@ Add this to the `:dependencies` stanza of your `project.clj`:
 
 `uap-clj` depends on the file `regexes.yaml` actively maintained in the public [`ua-parser/uap-core`](https://github.com/ua-parser/uap-core) repository, as well as on the test fixtures `test_ua.yaml`, `test_os.yaml`, and `test_device.yaml` contained therein. Be sure to run `rm -rf .lein-get-deps && lein git-deps && lein deps` after cloning this code repository, and re-run on occasion to pull in changes committed to those `uap-core` assets.
 
+`uap-clj` needs `regexes.yaml` in `edn` format. One way do this conversion is to run:
+
+``` bash
+uap-clj$ cat src/resources/submodules/regexes.yaml | jet -i yaml -o edn -k > resources/regexes.edn
+```
+To get [jet](https://github.com/borkdude/jet).
```
Comment (russellwhitaker) [inline README.md]: Suggest:
Install [jet](https://github.com/borkdude/jet) CLI:
```bash
brew install borkdude/brew/jet
```

File: README.md [hunk 2]
```diff
@@ -18,6 +18,15 @@ Add this to the `:dependencies` stanza of your `project.clj`:
...
+
+This removes the dependency on `clj-yaml` during runtime and replaces it with and dependency on `edn` which is built in.
```
Comment (russellwhitaker) [inline README.md]: Suggest:

Converting the `regexes.yaml` file to native Clojure EDN data format removes the runtime `clj-yaml` dependency.

**Review by russellwhitaker — CHANGES_REQUESTED** [review]

**Comment by russellwhitaker** [comment]
Thanks very much for your contribution @vigneshsarma . This work eliminates future risk (in runtime) of picking up more `snakeyaml` CVE vulnerabilities (e.g. CVE-2022-1471).

**Pull request merged**